precompiles/nfts/swap/src/tests.rs

```diff
@@ -33,15 +63,94 @@ fn selectors() {
 
 #[test]
 fn create_swap_works() {
-	todo!()
+	ExtBuilder::<Runtime>::default()
+        .with_balances(vec![(ALICE.into(), 1000), (BOB.into(), 1000)])
+        .build_with_collections()
+        .execute_with(|| {
+			mint_each_collections();
+
+			assert_eq!(get_owner_of_item(0, 1), Some(ALICE.into()));
+			_precompiles().prepare_test(
+				ALICE,
+				Precompile1,
+				PCall::create_swap {
+					offered_collection: 0.into(),
+					offered_item: 1.into(),
+					desired_collection: 1.into(),
+					maybe_desired_item: OptionalU256::from(Some(U256::from(1))),
+					maybe_price: OptionalPriceWithDirection::from(Some(PriceWithDirection::new(U256::from(100), PriceDirection::Receive))),
+					duration: 100.into(),
+				},
+			).execute_returns(true);
+			assert_eq!(get_owner_of_item(0, 1), Some(ALICE.into()));
```

Comment:
This assert is kinda useless as it doesn't verify that the swap was created, it only do a check that was already done before the precompile call already